src/Libraries/RevitNodes/Elements/Element.cs

```diff
@@ -212,7 +212,7 @@ public Element ElementType
                 else
```

Comment:
let's remove the `else`, no need to wrap the code in the `else` since the `if` returns.